### ISSUE TITLE
 Upgrade sqlx dependency

### ISSUE THREAD

**Issue by grunch** [MostroP2P/mostro#166]
Upgrade sqlx to latest version

**Comment by f3r10** [comment]
hi @grunch as a PR I am trying to update the sqlx version but it depends that `sqlx-crud` also updates to the latest version of sqlx :sweat_smile: 

.. btw I cloned `sqlx-crud` and after updating the sqlx version and some tweaks on `mostro` and `mostro-core`, I was able to update to the latest version of sqlx.

**Comment by grunch** [comment]
wow that's great, I remember that we broke some things after try to update, I don't remember exactly but @arkanoider was working on it

**Comment by arkanoider** [comment]
@f3r10 you are a hero doing this.
If I can help you ask freely...
I think sqlx crud just needs to be updated to latest.sqlx, but rhen we have to work on it and dev never answered!
Your commitment is super good!

**Comment by f3r10** [comment]
maybe until the maintainer of `sqlx-crud` bumps the version of `sqlx`, it would be possible to create a fork of the project and update the reference of the dependency like so
```toml
sqlx-crud = { git = "https://github.com/f3r10/sqlx-crud", features = ["runtime-tokio-rustls"] }
```

**Comment by arkanoider** [comment]
Yep! I think he dropped it, we should takeover! Great to have a volunteer...

**Comment by f3r10** [comment]
I think  the first step for upgrading the sqlx would be in mostro-core, so I created a pull request about: https://github.com/MostroP2P/mostro-core/pull/57

**Comment by arkanoider** [comment]
> I think the first step for upgrading sqlx version would be in mostro-core, so I created a pull request about: [MostroP2P/mostro-core#57](https://github.com/MostroP2P/mostro-core/pull/57)

This is the starting point for sure, it's passed some time. Now I think if I remember well that also mostro needs to update `sqlx` otherwise we will have some errors. Test it please...you need to bump update on Mostro and see if it compiles correctly and work.
Before merging this we have to be sure to do not break all, then to maintain it i had started to study the crate, which is a nice derivative macro! Hope you want to try to learn it...one nice thing should be having also blocking calls to CRUD functions for example...

**Comment by f3r10** [comment]
yes, you are right it would also be necessary to bump the version of `sqlx` and `sqlx-crud` on `Mostro`. I have already tested it and `Mostro` runs correctly.

I have not worked with macros on Rust, but it would be nice to learn it.

**Comment by juanramonperez** [comment]
Assigning this to me

**Comment by grunch** [comment]
@juanramonperez are you still working on this issue?

**Comment by juanramonperez** [comment]
@grunch I'm sorry for the late response, I'm not working on this by the moment, feel free to reassing if this is a priority

**Comment by SIDHARTH20K4** [comment]
I would like to work on this if if no one does!
@grunch

**Comment by grunch** [comment]
@SIDHARTH20K4 thank you, this issue was already addressed, we are using sqlx v0.9.0 now, closing this one
